Content.Server/Strip/StrippableSystem.cs

```diff
@@ -305,7 +309,7 @@ bool Check()
 
             if (Check())
             {
-                if (userHands.ActiveHandEntity != null)
+                if (userHands.ActiveHandEntity != null && !ev.Stealth)
```

Comment:
As below, you can avoid like 5 if statements for something you already know.

Reply:
Done